lib/index.js

```diff
@@ -32,8 +32,23 @@ const internals = {
         health: {
             path: ['/health', '/healthcheck'],
             response: {
-                ver: process.env.npm_package_version,
-                sha: process.env.VCS_REF
+                ver: process.env.npm_package_version ? process.env.npm_package_version : '0.0.0',
+                sha: process.env.VCS_REF,
```

Comment:
we can either add a default per field, ie:

```suggestion
                sha: process.env.VCS_REF ? process.env.VCS_REF : '000000',
```

or do something like `internals.defaults = { ver: 'foo', sha: 'bar' }` and then `merge` the object with whatever's passed when initializing this plugin, that way, we override any missing fields with sane defaults. I think I'm leaning a bit more for the latter.